<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CATS2</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>3</number>
    </property>
    <property name="topMargin">
     <number>3</number>
    </property>
    <property name="rightMargin">
     <number>3</number>
    </property>
    <property name="bottomMargin">
     <number>3</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="rightMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item>
        <widget class="QTabWidget" name="trackingSettingsWidget">
         <property name="currentIndex">
          <number>-1</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="robotsControllerWidget" native="true"/>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QWidget" name="secondaryViewerWidget" native="true"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QWidget" name="primaryViewerWidget" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>586</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionZoomIn"/>
    <addaction name="actionZoomOut"/>
    <addaction name="actionAdjustView"/>
    <addaction name="separator"/>
    <addaction name="actionSaveCurrentView"/>
    <addaction name="separator"/>
    <addaction name="actionShowAgentsData"/>
    <addaction name="actionShowControlAreas"/>
    <addaction name="separator"/>
    <addaction name="actionShowSetupOutline"/>
   </widget>
   <widget class="QMenu" name="menuRobots">
    <property name="title">
     <string>Robots</string>
    </property>
    <addaction name="actionReconnectToRobots"/>
   </widget>
   <widget class="QMenu" name="menuFiles">
    <property name="title">
     <string>Files</string>
    </property>
    <addaction name="actionPreferences"/>
   </widget>
   <addaction name="menuFiles"/>
   <addaction name="menuView"/>
   <addaction name="menuRobots"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionZoomIn"/>
   <addaction name="actionZoomOut"/>
   <addaction name="actionAdjustView"/>
   <addaction name="separator"/>
   <addaction name="actionSaveCurrentView"/>
   <addaction name="separator"/>
   <addaction name="actionShowAgentsData"/>
   <addaction name="actionShowControlAreas"/>
   <addaction name="separator"/>
   <addaction name="actionShowSetupOutline"/>
   <addaction name="separator"/>
   <addaction name="actionReconnectToRobots"/>
  </widget>
  <action name="actionZoomIn">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/resources/icons/ic_zoom_in_black_24dp.png</normaloff>:/resources/icons/ic_zoom_in_black_24dp.png</iconset>
   </property>
   <property name="text">
    <string>Zoom in</string>
   </property>
   <property name="toolTip">
    <string>Zoom in the image</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/resources/icons/ic_zoom_out_black_24dp.png</normaloff>:/resources/icons/ic_zoom_out_black_24dp.png</iconset>
   </property>
   <property name="text">
    <string>Zoom out</string>
   </property>
   <property name="toolTip">
    <string>Zoom out the image</string>
   </property>
  </action>
  <action name="actionSaveCurrentView">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/resources/icons/ic_camera_enhance_black_24dp.png</normaloff>:/resources/icons/ic_camera_enhance_black_24dp.png</iconset>
   </property>
   <property name="text">
    <string>Save current view</string>
   </property>
  </action>
  <action name="actionAdjustView">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/resources/icons/ic_zoom_out_map_black_24dp.png</normaloff>:/resources/icons/ic_zoom_out_map_black_24dp.png</iconset>
   </property>
   <property name="text">
    <string>AdjustView</string>
   </property>
   <property name="toolTip">
    <string>Adjusts the view to the available area</string>
   </property>
  </action>
  <action name="actionShowAgentsData">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/resources/icons/ic_near_me_black_24dp.png</normaloff>:/resources/icons/ic_near_me_black_24dp.png</iconset>
   </property>
   <property name="text">
    <string>Show agents data on the scene</string>
   </property>
   <property name="toolTip">
    <string>Show agents data on the scene</string>
   </property>
  </action>
  <action name="actionShowControlAreas">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/resources/icons/ic_dashboard_black_24dp.png</normaloff>:/resources/icons/ic_dashboard_black_24dp.png</iconset>
   </property>
   <property name="text">
    <string>Show control areas for selected robot</string>
   </property>
   <property name="toolTip">
    <string>Show control areas for selected robot</string>
   </property>
  </action>
  <action name="actionShowSetupOutline">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/resources/icons/ic_grid_on_black_24dp.png</normaloff>:/resources/icons/ic_grid_on_black_24dp.png</iconset>
   </property>
   <property name="text">
    <string>Show setup outline</string>
   </property>
   <property name="toolTip">
    <string>Show setup outline</string>
   </property>
  </action>
  <action name="actionReconnectToRobots">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/resources/icons/ic_autorenew_black_24dp.png</normaloff>:/resources/icons/ic_autorenew_black_24dp.png</iconset>
   </property>
   <property name="text">
    <string>Re-establish the connection with robots</string>
   </property>
   <property name="toolTip">
    <string>Re-establish the connection with robots</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/resources/icons/ic_settings_black_24dp.png</normaloff>:/resources/icons/ic_settings_black_24dp.png</iconset>
   </property>
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections/>
</ui>
